web/react/components/unread_channel_indicator.jsx

```diff
@@ -31,5 +31,5 @@ UnreadChannelIndicator.defaultProps = {
 UnreadChannelIndicator.propTypes = {
     show: React.PropTypes.bool,
     extraClass: React.PropTypes.string,
-    text: React.PropTypes.string
+    text: React.PropTypes.object
```

Comment:
This is fine as it is, but for future reference, if you're passing a React element as a prop, you can use React.PropTypes.node to be a bit more specific than using a plain object.